src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -754,6 +756,7 @@ namespace OpenLoco::Audio
             return;
         }
 
+        // Check if vehicle appears in any other (windowed) viewport
         for (auto i = (int32_t)WindowManager::count() - 1; i >= 0; i--)
```

Comment:
This loop is rather bizarre, why does it have to go backwards? It doesn't seem to care about the order, as long a viewport has the location visible it bails so it shouldn't matter.

Reply:
Guess it's a micro-optimisation, as you'll skip the main window and toolbars this way.

Reply:
I think its cache friendlier to walk forwards and not backwards even if it skips a few iterations.

Reply:
Yeah, it is odd, especially given it's not even modifying the window list, right. (We don't have a 'dead' window flag yet, so that is something to look out for, still.) But even now, we could probably even refactor it all to be just one loop, handling the main window as a special case (due to the quarter zoom business). That's getting a bit out of scope for this PR, though.

Reply:
Yes, just a remark, if I insist on changes I will request it explicitly as a review.